multiwallet.py

```diff
@@ -469,6 +493,60 @@ def do_generate_seed(self, arg):
             ),
         )
 
+    def do_blind_xpub(self, arg):
+        """Blind an XPUB using a random BIP32 path. Experts only!"""
+
+        # Prevent footgun-ing
+        if not self.ADVANCED_MODE:
+            warning_msg = (
+                "Blinding an xpub must be performed correctly or you could lose funds!\n"
+                "Multiwallet supports the custom BIP32 paths needed, but few HWWs can sign on these paths.\n"
+                "Some can't even co-sign a multisig transaction with nonstandard BIP32 paths (even if their path is standard)"
+            )
+            _print_footgun_warning(warning_msg)
+            return
+
+        key_record_dict = _get_key_record()
+        xfp_hex = key_record_dict["xfp"]
```

Comment:
what does xfp stand for? isn't something like `fingerprint` easier to read?

Reply:
I believe it's something that coldcard invented and the x is to signify that it's the root fingerprint (because you can't verify a hardened child fingerprint from an xpub).

Happy to change it to something else if you prefer? If we do that, it should be throughout the codebase as it's something I put in quite a bit (62 occurrences by my count):
```
$ ag xfp | wc -l
      62
```

Reply:
Added a github issue: https://github.com/buidl-bitcoin/buidl-python/issues/54